### PR TITLE
Replaced deprecated entitymanager

### PR DESCRIPTION
**Pull request by matslats**
4 instances replaced with entityFieldManager

### CODE REVIEW

**Comment by RoySegall** [comment]
It seems that there is no entity_field.manager service - https://travis-ci.org/amitaibu/og/jobs/103112597#L203.

File: src/OgAccess.php [hunk 1]
```diff
@@ -191,7 +191,8 @@ public static function userAccessEntity($operation, EntityInterface $entity, Acc
       foreach ($entity_groups as $groups) {
         foreach ($groups as $group) {
           $user_access = static::userAccess($group, $operation, $user);
-          if ($user_access->isAllowed()) {
+          if (!$user_access->isForbidden()) {
+            //this covers allowed and neutral
```
Comment (pfrenssen) [inline src/OgAccess.php:195]: This commit is from PR #133 - it crept in here accidentally. Best is to rebase the branch.

File: tests/src/Unit/OgAccessEntityTestBase.php [hunk 2]
```diff
@@ -44,9 +44,9 @@ public function setup() {
 
     $this->groupManager->isGroup($entity_type_id, $bundle)->willReturn(FALSE);
 
-    $entity_manager = $this->prophesize(EntityManagerInterface::class);
-    $entity_manager->getFieldDefinitions($entity_type_id, $bundle)->willReturn([$field_definition->reveal()]);
-    \Drupal::getContainer()->set('entity.manager', $entity_manager->reveal());
+    $entity_field_manager = $this->prophesize(EntityFieldManagerInterface::class);
+    $entity_field_manager->getFieldDefinitions($entity_type_id, $bundle)->willReturn([$field_definition->reveal()]);
+    \Drupal::getContainer()->set('entity.manager', $entity_field_manager->reveal());
```
Comment (pfrenssen) [inline tests/src/Unit/OgAccessEntityTestBase.php]: To fix the failures, try changing this line to

```
\Drupal::getContainer()->set('entity_field.manager', $entity_field_manager->reveal());
```

**Comment by pfrenssen** [comment]
@matslats, you've closed it but wasn't merged yet. We should get this in, this is a good improvement. The original branch was removed but I still have a copy locally. I'll rebase it to remove the stray commit from #133 and push it to the main repository so it can be reviewed.

**Comment by matslats** [comment]
I got confused and made another branch myself.
Then in my HEAD the changes were already made.
So I didn't know how to commit the difference.

**Comment by pfrenssen** [comment]
I can tell you how I did it, so in case you encounter this again in the future you can solve it. Note that I hadn't downloaded your last commit `90ea16d` so I recreated it manually.

```
# Recreate the missing branch from the commit that was still in my local repo.
# I create a new branch, and then "reset" it to the commit number.
$ git checkout -b replaced-deprecated-entitymanager
$ git reset --hard 92a815e

# Then use interactive rebase to remove the wrong commit.
$ git rebase -i origin/8.x-1.x

# Finally, push the branch.
$ git push origin replaced-deprecated-entitymanager -u
```

**Pull request closed without merging**